roles/wordpress-install/tasks/main.yml

```diff
@@ -37,6 +37,27 @@
   when: item.value.site_install == True and (item.value.multisite.enabled | default(False) == False)
   changed_when: "'WordPress is already installed.' not in wp_install_results.stdout"
 
+- name: Check Existing Permalink Structure
+  command: wp rewrite list --format=count
+           --allow-root
+           --url="{{ item.value.env.wp_home }}"
+  args:
+    chdir: "{{ www_root }}/{{ item.key }}/current/"
+  register: wp_permalink_check_results
+  with_dict: wordpress_sites
+  when: item.value.site_install and not item.value.multisite.enabled | default(false)
+  changed_when: False
+
+- name: Setup Permalink Structure
+  command: wp rewrite structure {{ item.value.permalink_structure | default("/%postname%/") }}
+           --allow-root
+           --url="{{ item.value.env.wp_home }}"
+  args:
+    chdir: "{{ www_root }}/{{ item.key }}/current/"
+  register: wp_permalink_results
+  with_dict: wordpress_sites
+  when: "'No rewrite rules' in wp_permalink_check_results.results.0.stderr"
```

Comment:
@fullyint ah, I did not see those other PRs.  I'll let those handle it.

Also, it was actually you who simplified the conditionals :) I just copied your suggestion from the first comment here.  Previously, I was just trying to follow the same format (at the time) as the other tasks.

I'll stay out of this kind of detail and leave it up to you collaborators. 

Thanks for Trellis and I will keep an eye out for future issues I think I could help solve!